crates/maple_cli/src/cmd/tagfiles.rs

```diff
@@ -19,40 +23,61 @@ struct TagInfo {
 }
 
 impl TagInfo {
-    pub fn format(&self, winwidth: usize) -> String {
+    pub fn format(&self, cwd: &PathBuf, winwidth: usize) -> String {
         let name = format!("{} ", self.name);
         let taken_width = name.len() + 1;
         let path_len = self.path.len() + 2;
         let mut adjustment = 0;
-        let path = if taken_width > winwidth {
-            format!("[{}]", self.path)
+
+        let mut home_path = PathBuf::new();
+        let path = Path::new(&self.path);
+        let path = path.strip_prefix(cwd).unwrap_or(
+            // FIXME: is there a way to avoid cloning HOME?
```

Comment:
@ImmemorConsultrixContrarie if you have any hints here (or on the PR in general) I'd be happy to have them :)

Reply:
`HOME.as_deref().map()`